XmlSchemas/ConfigurationOverrides/overrides.xsd

```diff
@@ -4,29 +4,102 @@
     <xsd:complexType>
       <xsd:sequence>
         <xsd:element minOccurs="0" maxOccurs="unbounded" name="Service">
+          <xsd:annotation>
+            <xsd:documentation>AWS Tools for PowerShell configuration for a service.</xsd:documentation>
+          </xsd:annotation>
           <xsd:complexType>
             <xsd:all>
-              <xsd:element name="FileVersion" type="xsd:int" />
-              <xsd:element minOccurs="0" name="SkipCmdletGeneration" type="xsd:boolean" />
-              <xsd:element name="C2jFilename" type="xsd:string" />
-              <xsd:element minOccurs="0" name="AssemblyName" type="xsd:string" />
-              <xsd:element minOccurs="0" name="ServiceNounPrefix" type="xsd:string" />
-              <xsd:element minOccurs="0" name="ServiceName" type="xsd:string" />
-              <xsd:element minOccurs="0" name="ServiceClientInterface" type="xsd:string" />
-              <xsd:element minOccurs="0" name="ServiceClient" type="xsd:string" />
-              <xsd:element minOccurs="0" name="ServiceModuleGuid" type="xsd:string" />
-              <xsd:element minOccurs="0" name="DefaultRegion" type="xsd:string" />
-              <xsd:element minOccurs="0" name="PipelineParameter" type="xsd:string" />
+              <xsd:element name="FileVersion" type="xsd:int">
+                 <xsd:annotation>
+                    <xsd:documentation>ADVANCED CONFIGURATION. Current version of this configuration file. Defaults to 0 for services with older configurations. This is incremented during manual updates to the configuration of a service in order to forbid release automation to use older configuration overrides.</xsd:documentation>
+                </xsd:annotation>
+              </xsd:element>
+              <xsd:element minOccurs="0" name="SkipCmdletGeneration" type="xsd:boolean">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. Defaults to false. Set to true to disable generation of cmdlets for this service.</xsd:documentation>
+                </xsd:annotation>
+              </xsd:element>
+              <xsd:element name="C2jFilename" type="xsd:string">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. Name of the C2J file for this service (whithout extension). This is part of the initial configuration for the service and should't be changed after release.</xsd:documentation>
+                </xsd:annotation>
+              </xsd:element>
+              <xsd:element minOccurs="0" name="AssemblyName" type="xsd:string">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. Name of the AWS SDK for .NET assembly/package/namespace for the service (not including extension and AWSSDK prefix). This is part of the initial configuration for the service and should't be changed after release.</xsd:documentation>
```

Comment:
nit: ()s not needed and reads better without

Reply:
ok